plugins/modules/ec2_vpc_egress_igw_info.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+---
+module: ec2_vpc_egress_igw_info
```

Comment:
I would probably suggest adding this new module directly into amazon.aws rather than here.

Reply:
@alinabuzachis the new module has been removed and will be introduced with the migration of `ec2_vpc_egress_igw` to `amazon.aws`